S7.Net.UnitTest/S7NetTestsAsync.cs

```diff
@@ -883,16 +876,16 @@ public async Task Test_Async_ReadClassWithArrayAndCustomType()
             Assert.AreEqual(tc.CustomTypes[1].Bools[1], tc2.CustomTypes[1].Bools[1]);
         }
 
-        [TestMethod]
-        public async Task Test_Async_ReadWriteDouble()
-        {
-            double test_value = 55.66;
-            await plc.WriteAsync("DB1.DBD0", test_value);
-            var helper = await plc.ReadAsync("DB1.DBD0");
-            double test_value2 = Conversion.ConvertToDouble((uint)helper);
+        //[TestMethod]
```

Comment:
Let's delete this, git will remember anyway.

Reply:
I probably thought about adding long-word support to bring this test back. But I don't really see any value in word to other type conversions anyway, so can just remove it completely.